CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
+## 1.0.0 - 2015-11-18
+
+### Changed
+ - .sortable changed to a button to address accessibility
+ - Updated docs to note button preference for .sortable
+ - CSS properties re-ordered to better meet standards
```

Comment:
Incredibly trivial, but if you want to be consistent with the other lists in this file, you could remove the spaces before the hyphens :)